recipes/guiqwt/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "guiqwt" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fcad0c1745c2685d6dcc3367ba6c5fa5dbbcd5991bf005e787c36e8e49b520a2
+  fn: {{ name }}-{{ version }}.tar.gz
+
+  patches:
+    # Needed to avoid some random exceptions which can make the application
+    # crash
+    - curve.patch
+
+build:
+  number: 1000
+
+requirements:
+  host:
+    - setuptools
+    - python>=3
+    - qt=4.8.7
+    - pyqt=4.11.4
+    - pythonqwt
+    - guidata
+    - numpy
+    - cython
+    - scipy
+    - pillow
+    - vs2008_runtime  # [win]
```

Comment:
Please remove it

Reply:
I tried to put it there due to the win failure. It fails at resolving the metadata and I have no idea why